app/models/page.rb

```diff
@@ -124,8 +125,16 @@ def ensure_domain_tags
     self.add_tag("2l-domain:#{second_level_domain}")
   end
 
+  def ensure_news_tags
+    self.add_tag('news') if news
+  end
+
   protected
 
+  def news
```

Comment:
It would be a little more ruby-esque to name a boolean method like this with a question mark:

```suggestion
  def news?
```

Reply:
fixed.